images/scripts/services.setup

```diff
@@ -22,6 +22,9 @@ systemctl enable --now docker
 #
 #############
 
+# see https://github.com/freeipa/freeipa-container/issues/348
+rm /usr/libexec/oci/hooks.d/oci-systemd-hook
```

Comment:
Seems a bit cleaner to uninstall the package, but *shrug*, good enough.

Reply:
Uninstalling the package takes all of docker with it.